rift/data/models/target.py

```diff
@@ -96,6 +100,11 @@ def get_target(cls, tenant_id, target_id):
             object_name=TARGET_COLLECTION,
             query_filter={"id": target_id})
 
+        if not target_dict:
+            target_dict = db_handler.get_document(
+                object_name=TARGET_COLLECTION,
+                query_filter={"name": target_id})
```

Comment:
I'll merge this, but I'm wondering if there could be a better way of doing this. Is there a way to have an "or" in that query_filter?